.travis_scripts/openapi-validator.sh

```diff
@@ -2,6 +2,9 @@
 set -eu
 set -o pipefail
 
+curl -L https://raw.githubusercontent.com/OpenAPITools/openapi-generator/master/bin/utils/openapi-generator-cli.sh > ./openapi-generator-cli
+chmod +x ./openapi-generator-cli
+
```

Comment:
Shall we add an `echo` here to indicate the curl download is completed?

Reply:
@bzwei Added an `echo`.  Is that what you were looking for?